shared/constants.ts

```diff
@@ -17,3 +17,9 @@ export const ACTIVITY_LOG_STORE_KEY = 'activityLog';
 export const SETTINGS_STORE_KEY = 'settings';
 
 export const ONBOARDING_STATUS_STORE_KEY = 'isOnboardingDone';
+
+// TODO: move API key
+export const GNM_API_KEY = 'AIzaSyBt_ne78gyEk5AAsheh6q6TZAgAy1ncFVE';
```

Comment:
@rjmangubat23 need your help on how to secure this.

@kneckinator which account do we create an API key from? I'm using my Newlogic account for this.

Reply:
@pmigueld yes sure. I was thinking we can secure this via secrets then access it via a dotenv? @kneckinator what do you think?

Reply:
I agree with @rjmangubat23 's approach. We secure this using GitHub secrets.
@pmigueld can you revoke that key please and create one that we can put in secrets?
@jeremi can you please advise which Google account we use to generate the API key?

Reply:
Dotenv added and API key is now in GitHub secrets (`secrets.ENV_FILE`). Key was generated from my Google  account for now. Let me know if we will be using a different one.

Reply:
Let go for this. We merge this, then we raise bugs for the build process if needed as this PR is entangled with GitHub Action changes.